pyslurm/pyslurm.pyx

```diff
@@ -4854,89 +4880,201 @@ cdef class slurmdb_jobs:
 
         listNum = slurm.slurm_list_count(JOBSList)
         iters = slurm.slurm_list_iterator_create(JOBSList)
+        exit_status = 0
+        term_sig = 0
 
         for i in range(listNum):
             job = <slurm.slurmdb_job_rec_t *>slurm.slurm_list_next(iters)
 
             JOBS_info = {}
             if job is not NULL:
                 jobid = job.jobid
-                JOBS_info[u'account'] = slurm.stringOrNone(job.account, '')
-                JOBS_info[u'allocated_gres'] = slurm.stringOrNone(job.alloc_gres, '')
+                JOBS_info[u'account'] = noneToAnother(slurm.stringOrNone(job.account, ''), '')
```

Comment:
@dinesh121991 What's the purpose of `noneToAnother`?

Reply:
noneToAnother will convert None type to specific string. This allows to test the attributes in nosetest scripts, otherwise it does not match and always fail in those cases. This reduce unnecessary codes in the testscript and who all are going to use the library.